.prettierrc.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   bracketSpacing: true,
   jsxBracketSameLine: false,
   singleQuote: true,
-  trailingComma: 'all',
-};
\ No newline at end of file
+  trailingComma: 'es5',
```

Comment:
changed it to align with the Satya's ESLint plugin. Otherwise, the VSCode ESLint extension freaks out.